include/envoy/stream_info/stream_info.h

```diff
@@ -201,6 +201,9 @@ class StreamInfo {
    */
   virtual void onUpstreamHostSelected(Upstream::HostDescriptionConstSharedPtr host) PURE;
 
+  virtual void setRouteName(std::string name) PURE;
+
+  virtual const std::string& getRouteName() const PURE;
```

Comment:
Please add comments for these, since they're part of the public API.

Reply:
Will do